src/main/java/BlueTurtle/TSE/JSONFormatter.java

```diff
@@ -33,9 +34,22 @@
 	 */
 	public void format() throws IOException {
 		List<Warning> totalWarnings = new ArrayList<Warning>();
-		totalWarnings.addAll(parseCheckStyleXML());
-		totalWarnings.addAll(parsePMDXML());
-		totalWarnings.addAll(parseFindBugsXML());
+		
+		File file = new File(JavaController.getCheckStyleOutputFile());
+		if(file.exists() && !file.isDirectory()) { 
```

Comment:
Youre right about isDirectory, Ill remove it. How would you want me to change this into one method? As far as I see the code can't really be shortened.